Source/GUI/MemWatcher/MemWatchWidget.h

```diff
@@ -49,6 +49,7 @@ class MemWatchWidget : public QWidget
   bool warnIfUnsavedChanges();
   void restoreWatchModel(const QString& json);
   QString saveWatchModel();
+  QString m_watchListFile = "";
```

Comment:
```suggestion
  QString m_watchListFile;
```

Reply:
Is this safe to do? Is an uninitialized QString implicitly ""?

Reply:
As per [`QString::QString()`](https://doc.qt.io/qt-6/qstring.html#QString), it is safe.

Reply:
Done